content/source/docs/extend/best-practices/sensitive-state.html.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: "extend"
+page_title: "Extending Terraform: Sensitive State Best Practices"
+sidebar_current: "docs-extend-best-practices-sensitive-state"
+description: |-
+  Recommendations for handling sensitive information in state.
+---
+
+# Handling Sensitive Values in State
+
+Many organizations use Terraform to manage their entire infrastructure, and
+it's inevitable that [sensitive information will find its way into
+Terraform](/docs/state/sensitive-data.html) in these circumstances. There are a
+couple of recommended approaches for managing sensitive state in Terraform.
+
+## Using the `Sensitive` Flag
+
+When working with a field that is likely to contain sensitive information, it
```

Comment:
Nitpick, but I'd reword this as _field that contains information that is likely to be considered as sensitive_. The reason being that I'd not want anyone to use this sentence as an excuse to tuck `Sensitive` to all fields "just in case".